code/JTWATTS/PYTHON/Persons_Class.py

```diff
@@ -0,0 +1,32 @@
+from turtle import clear
+
+
+class Person:
+    
+    def __init__(self,name,age):
+        
+        self.age = age
+        self.name = name
+    
+       
+
+    def display(self):
+           print(self.age,self.name)
+           return self.display
+    def display_student(self):
+            print(self.age,self.name,self)
+            return self.display_students
+
+
+Student=Person(12,'jack')
```

Comment:
You're missing the implementation of the Class Student. The Class Student should inherit from the class person. This is the formula:

class Rabbit(Animal):
    def __init__(self, legs, color, speed): ##adding the new attribute speed
        super().__init__(legs, color) ## referencing old attributes legs,color
        self.speed = speed
        
  You can review inheritance [here  ](https://github.com/PdxCodeGuild/class_chihuahua/blob/main/1%20Python/docs/OOP.md)